functions/HttpTriggerAPIUsersIdTaskId/__init__.py

```diff
@@ -6,71 +10,124 @@
 #GET API method function
 def get(param1, param2):

```

Comment:
Does it make sense to just have the method names be get, update, etc... or should we be doing something like get_user_task, update_user_task, etc (something more descriptive)? Maybe it's obvious given the file that it's in?

Reply:
That's exactly it, my approach was that looking at the parent folder name aka API endpoint makes it clear what those CRUD functions methods are for